crypto/crypto.go

```diff
@@ -111,6 +122,8 @@ func (client *Crypto) Configure(config core.ServerConfig) error {
 		return client.setupFSBackend(config)
 	case "vaultkv":
 		return client.setupVaultBackend(config)
+	case "storageapi":
```

Comment:
could it just be a URL instead, and check if it starts with `<something>://`? I find `storageapi` a weird backend to configure.

Or if we want to have a config struct for future extensibility (although that could still be the case), maybe rename it to `secure-store` (also extra indicates the others aren't considered secure) or `external-store` / `external`.

Reply:
I don't think we should use a url there, but I agree with the storageapi name. I opted to `external-store`.